rpi-eeprom-update

```diff
@@ -320,25 +440,26 @@ removePreviousUpdates()
       # N.B. recovery.bin is normally ignored by the ROM if is not a valid
       # executable but it's best to not have the file at all.
       rm -f "${BOOTFS}/recovery.bin"
-      rm -f "${BOOTFS}/pieeprom.bin" "${BOOTFS}/pieeprom.upd"
+      rm -f "${BOOTFS}/pieeprom.bin" "${BOOTFS}/pieeprom.upd" "${BOOTFS}/pieeprom.sig"
+      rm -f "${BOOTFS}/vl805.bin" "${BOOTFS}/vl805.sig"
       # Case insensitive for FAT bootfs
-      find "${BOOTFS}" -maxdepth 1 -type f -iname "recovery.*" -regex ".*\.[0-9][0-9][0-9]$" -exec rm -f {} \;
+      find "${BOOTFS}" -maxdepth 1 -type f -iname "recovery.*" -regex '.*\.[0-9][0-9][0-9]$' -exec rm -f {} \;
    fi
 }
 
 checkVersion()
 {
-   getCurrentVersion
-   getUpdateVersion
-   if [ "${UPDATE_VERSION}" -gt "${CURRENT_VERSION}" ]; then
+   lookupVersionInfo
+
+   if [ "${BOOTLOADER_UPDATE_VERSION}" -gt "${BOOTLOADER_CURRENT_VERSION}" ]; then
       echo "*** UPDATE REQUIRED ***"
-      printVersions "${CURRENT_VERSION}" "${UPDATE_VERSION}"
-      write_status_info EXIT_UPDATE_REQUIRED "${CURRENT_VERSION}" "${UPDATE_VERSION}"
+      printVersions
+      write_status_info "${EXIT_UPDATE_REQUIRED}"
```

Comment:
Did you really mean to change this to integers? I guess so, and now the comment on your previous commit makes sense, however the change `${1:-EXIT_FAILED}` does not as that will output a string if $1 is not defined. I can submit a PR if you let me know which you prefer - strings kinda made sense as if you decide to change the internal representation in future it doesn't affect the machine output (unless you change the strings, of course...)